snuba/query/snql/parser.py

```diff
@@ -672,9 +678,14 @@ def visit_select_clause(
         return selected_columns
 
     def visit_selected_expression(
-        self, node: Node, visited_children: Tuple[Any, Expression]
+        self,
+        node: Node,
+        visited_children: Tuple[Any, Union[SelectedExpression, Expression]],
     ) -> SelectedExpression:
         _, exp = visited_children
+        if isinstance(exp, SelectedExpression):
+            return exp
+
         alias = exp.alias or node.text.strip()
```

Comment:
Can `exp.alias` ever be anything but None at this point ?

Reply:
Yes, if `exp` is a function with an alias.